airflow/providers/google/cloud/example_dags/example_dataproc.py

```diff
@@ -178,13 +179,26 @@
     },
     "jobs": [{"step_id": "pig_job_1", "pig_job": PIG_JOB["pig_job"]}],
 }
-BATCH_ID = "test-batch-id"
+BATCH_ID = f"test-batch-id-{str(uuid4())}"
 BATCH_CONFIG = {
     "spark_batch": {
         "jar_file_uris": ["file:///usr/lib/spark/examples/jars/spark-examples.jar"],
         "main_class": "org.apache.spark.examples.SparkPi",
     },
 }
+BATCH_CONFIG_WITH_PHS = {
+    "spark_batch": {
+        "jar_file_uris": ["file:///usr/lib/spark/examples/jars/spark-examples.jar"],
+        "main_class": "org.apache.spark.examples.SparkPi",
+    },
+    "environment_config": {
+        "peripherals_config": {
+            "spark_history_server_config": {
+                "dataproc_cluster": f"projects/{PROJECT_ID}/regions/{REGION}/clusters/phs-cluster-name"
```

Comment:
Do we need to create cluster manually? What if I will take this example DAG and I will execute it? Operator which use this config will fail or pass?

Reply:
If you run this config, operator will be fail. I will try to create cluster automatically and then I will change this PR

Reply:
👍